addons/website/tests/test_page.py

```diff
@@ -303,3 +303,21 @@ def test_homepage_not_slash_url(self):
         root_html = html.fromstring(r.content)
         canonical_url = root_html.xpath('//link[@rel="canonical"]')[0].attrib['href']
         self.assertEqual(canonical_url, website.domain + "/")
+
+    def test_t_cache_footer_debug_link(self):
+        # Debug link (flag) should be use the current url.
+        self.url_open('/')
+        self.url_open('/contactus')
+        r = self.url_open('/?debug=1')
+        self.assertIn('<a class="o_debug_mode" href="?debug="', r.text)
+        r = self.url_open('/contactus?debug=1')
+        self.assertIn('<a class="o_debug_mode" href="?debug="', r.text)
+
+    def test_head_canonial_href(self):
+        # canonial href should be use the current url.
+        self.url_open('/')
+        self.url_open('/contactus')
+        r = self.url_open('/')
+        self.assertRegex(r.text, r'<link rel="canonical" href="http://[^/]+/"/>')
+        r = self.url_open('/contactus')
+        self.assertRegex(r.text, r'<link rel="canonical" href="http://[^/]+/contactus"/>')
```

Comment:
Don't we need multiple lang installed for the canonical tags to appear? I don't thinnk it is the case here but the branch is red.
I tested your test in regular master (without your fix) and test is also green.
Something seems wrong?